tidy3d/plugins/invdes/region.py

```diff
@@ -159,12 +175,69 @@ class TopologyDesignRegion(DesignRegion):
         "Supplying ``False`` will completely leave out the override structure.",
     )
 
+    def _validate_eps_values(self):
+        """Validate the epsilon values by evaluating the transformations."""
+        try:
+            x = self.initial_parameters
+            self.eps_values(x)
+        except Exception as e:
+            raise ValidationError(f"Could not evaluate transformations: {str(e)}") from e
```

Comment:
will this print `e` twice though?

Reply:
no

Reply:
I think how you have it is fine, I just get confused about exception handling.